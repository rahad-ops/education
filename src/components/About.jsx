import React from 'react';
import List, { ListItem, ListItemText } from 'material-ui/List';
import Avatar from 'material-ui/Avatar';
import ImageIcon from 'material-ui-icons/Image';
import WorkIcon from 'material-ui-icons/Work';
import BeachAccessIcon from 'material-ui-icons/BeachAccess';

const About = () => (
  <div>
    <h1>About Page</h1>
    <p>Did you get here via Redux?</p>
    <List>
      <ListItem>
        <Avatar>
          <ImageIcon />
        </Avatar>
        <ListItemText primary="Photos" secondary="Jan 9, 2014" />
      </ListItem>
      <ListItem>
        <Avatar>
          <WorkIcon />
        </Avatar>
        <ListItemText primary="Work" secondary="Jan 7, 2014" />
      </ListItem>
      <ListItem>
        <Avatar>
          <BeachAccessIcon />
        </Avatar>
        <ListItemText primary="Vacation" secondary="July 20, 2014" />
      </ListItem>
    </List>
  </div>
);

export default About;
