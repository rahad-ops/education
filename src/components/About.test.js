import React from 'react';
import About from './About';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

describe('About', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<About />, div);
  });

  test('has a valid snapshot', () => {
    const component = renderer.create(<About />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
