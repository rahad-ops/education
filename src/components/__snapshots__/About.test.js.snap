// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About has a valid snapshot 1`] = `
<div>
  <h1>
    About Page
  </h1>
  <p>
    Did you get here via Redux?
  </p>
  <ul
    className="MuiList-root-1 MuiList-padding-2"
  >
    <li
      className="MuiListItem-root-5 MuiListItem-default-8 MuiListItem-gutters-12"
      disabled={false}
    >
      <div
        className="MuiAvatar-root-15 MuiAvatar-colorDefault-16"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-18"
          color={undefined}
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M21 19V5c0-1.1-.9-2-2-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2zM8.5 13.5l2.5 3.01L14.5 12l4.5 6H5l3.5-4.5z"
          />
        </svg>
      </div>
      <div
        className="MuiListItemText-root-24"
      >
        <h3
          className="MuiTypography-root-30 MuiTypography-subheading-37 MuiListItemText-primary-27"
        >
          Photos
        </h3>
        <p
          className="MuiTypography-root-30 MuiTypography-body1-39 MuiTypography-colorTextSecondary-52 MuiListItemText-secondary-28"
        >
          Jan 9, 2014
        </p>
      </div>
    </li>
    <li
      className="MuiListItem-root-5 MuiListItem-default-8 MuiListItem-gutters-12"
      disabled={false}
    >
      <div
        className="MuiAvatar-root-15 MuiAvatar-colorDefault-16"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-18"
          color={undefined}
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M20 6h-4V4c0-1.11-.89-2-2-2h-4c-1.11 0-2 .89-2 2v2H4c-1.11 0-1.99.89-1.99 2L2 19c0 1.11.89 2 2 2h16c1.11 0 2-.89 2-2V8c0-1.11-.89-2-2-2zm-6 0h-4V4h4v2z"
          />
        </svg>
      </div>
      <div
        className="MuiListItemText-root-24"
      >
        <h3
          className="MuiTypography-root-30 MuiTypography-subheading-37 MuiListItemText-primary-27"
        >
          Work
        </h3>
        <p
          className="MuiTypography-root-30 MuiTypography-body1-39 MuiTypography-colorTextSecondary-52 MuiListItemText-secondary-28"
        >
          Jan 7, 2014
        </p>
      </div>
    </li>
    <li
      className="MuiListItem-root-5 MuiListItem-default-8 MuiListItem-gutters-12"
      disabled={false}
    >
      <div
        className="MuiAvatar-root-15 MuiAvatar-colorDefault-16"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-18"
          color={undefined}
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M13.127 14.56l1.43-1.43 6.44 6.443L19.57 21zm4.293-5.73l2.86-2.86c-3.95-3.95-10.35-3.96-14.3-.02 3.93-1.3 8.31-.25 11.44 2.88zM5.95 5.98c-3.94 3.95-3.93 10.35.02 14.3l2.86-2.86C5.7 14.29 4.65 9.91 5.95 5.98zm.02-.02l-.01.01c-.38 3.01 1.17 6.88 4.3 10.02l5.73-5.73c-3.13-3.13-7.01-4.68-10.02-4.3z"
          />
        </svg>
      </div>
      <div
        className="MuiListItemText-root-24"
      >
        <h3
          className="MuiTypography-root-30 MuiTypography-subheading-37 MuiListItemText-primary-27"
        >
          Vacation
        </h3>
        <p
          className="MuiTypography-root-30 MuiTypography-body1-39 MuiTypography-colorTextSecondary-52 MuiListItemText-secondary-28"
        >
          July 20, 2014
        </p>
      </div>
    </li>
  </ul>
</div>
`;
